core/engine/src/lib.rs

```diff
@@ -3,7 +3,7 @@
 //!
 //! # Example usage
 //!
-//! You can find multiple examples of the usage of Boa in the [`boa_examples`][examples] crate. In
```

Comment:
I think the name of the crate was correct because the crate is called `boa_examples` as can be seen in the [examples/Cargo.toml](https://github.com/boa-dev/boa/blob/main/examples/Cargo.toml#L2).